Source/JavaScriptCore/API/JSScript.h

```diff
@@ -84,22 +84,19 @@ JSC_CLASS_AVAILABLE(macos(10.15), ios(13.0))
 - (BOOL)cacheBytecodeWithError:(out NSError * _Nullable * _Nullable)error;
 
 /*!
- @method
  @abstract Returns true when evaluating this JSScript will use the bytecode cache. Returns false otherwise.
  */
-- (BOOL)isUsingBytecodeCache;
+@property (nonatomic, getter=isUsingBytecodeCache, readonly) BOOL usingBytecodeCache;
 
 /*!
- @method
  @abstract Returns the JSScriptType of this JSScript.
  */
-- (JSScriptType)type;
+@property (nonatomic, readonly) JSScriptType type;
 
 /*!
- @method
  @abstract Returns the sourceURL of this JSScript.
  */
-- (NSURL *)sourceURL;
+@property (nonatomic, readonly, copy) NSURL *_Nonnull sourceURL;
```

Comment:
This is already inside of a `NS_ASSUME_NONNULL_BEGIN`/`END` block. Is the `_Nonnull` really needed here?